spec/context-menu-manager-spec.coffee

```diff
@@ -156,3 +156,28 @@ describe "ContextMenuManager", ->
       catch error
         addError = error
       expect(addError.message).toContain('<>')
+
+    it "recursively applies templateForEvent on submenu items", ->
```

Comment:
This description is a little bit implementation-specific. How about something like

```coffee
it "calls `created` hooks for submenu items", ->
```